src/Files.Backend/CommandLine/ParsedCommands.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections.Generic;
+
+namespace Files.Backend.CommandLine
+{
+	public class ParsedCommands : List<ParsedCommand>
```

Comment:
Why is it empty?

Reply:
Already been empty

Reply:
Could you remove it and just replace with a `List<ParsedCommand>` in appropriate places?

Reply:
I’m unable to change because I cannot touch my PC for 2 weeks irl. This is a timer bomb lol so I believe we should merge as soon as possible. I can refactor in another PR, though.